Panel del círculo

FerGarci opened the pull request:
Primer versión del Panel del Círculo.

FerGarci committed:
Panel del círculo

UriMtzF commented on src/mx/uaemex/fi/paradigmas_I/figuras/vista/PanelCirculoData.java:
```diff
@@ -0,0 +1,45 @@
+package mx.uaemex.fi.paradigmas_I.figuras.vista;
+
+import mx.uaemex.fi.paradigmas_I.figuras.data.Figura;
```
Este import ya no es necesario, se puede dejar pero mejoraría el código quitarlo

UriMtzF commented on src/mx/uaemex/fi/paradigmas_I/figuras/vista/PanelCirculoData.java:
```diff
@@ -0,0 +1,45 @@
+    private JSpinner radio;
+
+    public PanelCirculoData(){
+        super();
```
Este método no es necesario, se puede omitir

FerGarci replied:
¿El super?

UriMtzF replied:
Sí, el super

UriMtzF commented on src/mx/uaemex/fi/paradigmas_I/figuras/vista/PanelCirculoData.java:
```diff
@@ -0,0 +1,45 @@
+
+        JLabel etqRadio;
+        SpinnerModel modelo;
+        JButton btnGD;
```
El botón no es necesario, este ya estará disponible en VentanaData

UriMtzF commented on src/mx/uaemex/fi/paradigmas_I/figuras/vista/PanelCirculoData.java:
```diff
@@ -0,0 +1,45 @@
+        add(radio);
+        add(btnGD);
+
+        setBorder(new EmptyBorder(10,10,10,10));
```
Este borde debería ser manejado en VentanaData por lo que no es necesario

UriMtzF commented on src/mx/uaemex/fi/paradigmas_I/figuras/vista/PanelCirculoData.java:
```diff
@@ -0,0 +1,45 @@
+        SpinnerModel modelo;
+        JButton btnGD;
+
+        setLayout(new GridLayout(3,1));
```
Cómo sobra el botón y para mantener estilos, es mejor hacer lo siguiente
```suggestion
        setLayout(new GridLayout(1,2));
```
De esta forma hay dos columnas, una para el label y otra para el spinner

UriMtzF commented on src/mx/uaemex/fi/paradigmas_I/figuras/vista/PanelCirculoData.java:
```diff
@@ -0,0 +1,45 @@
+        add(btnGD);
+
+        setBorder(new EmptyBorder(10,10,10,10));
+
```
Está bien si quedan espacios vacios, se puede estilizar luego, pero sería bueno que se eliminaran todas las líneas vacías

UriMtzF commented on src/mx/uaemex/fi/paradigmas_I/figuras/vista/PanelCirculoData.java:
```diff
@@ -0,0 +1,45 @@
+
+    }
+
+    @java.lang.Override
```
Esta es una anotación, a pesar de ser probablemente correcta es mejor escribirla de la siguiente forma:
```suggestion
    @Override
```

UriMtzF committed:
Actualizada anotación Override

FerGarci committed:
Panel del círculo actualizado

FerGarci committed:
Merge branch 'UriMtzF:master' into master

FerGarci committed:
Merge remote-tracking branch 'origin/master'

UriMtzF committed:
Eliminado .idea/misc.xml

Debido a la diferencia de JDK se elimina .idea/misc.xml sólo para aceptar el PR

UriMtzF commented on src/mx/uaemex/fi/paradigmas_I/figuras/vista/PanelCirculoData.java:
```diff
@@ -0,0 +1,35 @@
+    public void setData() {
+
+            }
+        }
```
Esta llave está desalineada pero no afecta el funcionamiento

UriMtzF committed:
Creado misc.xml

Creado misc.xml con correspondiente a proyecto principal

UriMtzF reviewed (approved):
El panel funciona correctamente
